types/simple-icons/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for simple-icons 2.5
+// Project: https://www.simpleicons.org
+// Definitions by: Eric Cornelisesn <https://github.com/ericcornelissen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface SimpleIcon {
+    title: string;
+    slug: string;
+    svg: string;
+    path: string;
+    source: string;
+    hex: string;
+}
+
+declare namespace SimpleIcons {
+    function get(name: string): SimpleIcon;
+}
+
+export default SimpleIcons;
```

Comment:
The library doesn't seem to be using default exports.
```suggestion
export = SimpleIcons;
```

Reply:
After changing it to this the linter told me to update it to :arrow_down: according to [export-just-namespace](https://github.com/Microsoft/dtslint/blob/master/docs/export-just-namespace.md):

```typescript
export function get(name: string): SimpleIcon;
```